tests/testthat/test-shinytest2-tm_g_barchart_simple.R

```diff
@@ -0,0 +1,346 @@
+  teal.data::datanames(data) <- datanames
+  teal.data::join_keys(data) <- teal.data::default_cdisc_join_keys[datanames]
+
+  init_teal_app_driver(
+    data = data,
+    modules = tm_g_barchart_simple(
+      label = "ADAE Analysis (e2e)",
+      x = teal.transform::data_extract_spec(
+        dataname = "ADSL",
+        select = teal.transform::select_spec(
+          choices = teal.transform::variable_choices(
+            "ADSL", c("ARM", "ACTARM", "SEX", "RACE", "ITTFL", "SAFFL", "STRATA2")
+          ),
+          selected = "ACTARM",
+          multiple = FALSE
+        )
+      ),
+      fill = list(
+        teal.transform::data_extract_spec(
+          dataname = "ADSL",
+          select = teal.transform::select_spec(
+            choices = teal.transform::variable_choices(
+              "ADSL", c("ARM", "ACTARM", "SEX", "RACE", "ITTFL", "SAFFL", "STRATA2")
+            ),
+            selected = "SEX",
+            multiple = FALSE
+          )
+        ),
+        teal.transform::data_extract_spec(
+          dataname = "ADAE",
+          select = teal.transform::select_spec(
+            choices = teal.transform::variable_choices("ADAE", c("AETOXGR", "AESEV", "AESER")),
+            selected = NULL,
+            multiple = FALSE
+          )
+        )
+      ),
+      x_facet = list(
+        teal.transform::data_extract_spec(
+          dataname = "ADAE",
+          select = teal.transform::select_spec(
+            choices = teal.transform::variable_choices("ADAE", c("AETOXGR", "AESEV", "AESER")),
+            selected = "AETOXGR",
+            multiple = FALSE
+          )
+        ),
+        teal.transform::data_extract_spec(
+          dataname = "ADSL",
+          select = teal.transform::select_spec(
+            choices = teal.transform::variable_choices(
+              "ADSL",
+              c("ARM", "ACTARM", "SEX", "RACE", "ITTFL", "SAFFL", "STRATA2")
+            ),
+            selected = NULL,
+            multiple = FALSE
+          )
+        )
+      ),
+      y_facet = list(
+        data_extract_spec(
+          dataname = "ADAE",
+          select = teal.transform::select_spec(
+            choices = teal.transform::variable_choices(
+              "ADAE",
+              c("AETOXGR", "AESEV", "AESER")
+            ),
+            selected = "AESEV",
+            multiple = FALSE
+          )
+        ),
+        data_extract_spec(
+          dataname = "ADSL",
+          select = teal.transform::select_spec(
+            choices = teal.transform::variable_choices(
+              "ADSL",
+              c("ARM", "ACTARM", "SEX", "RACE", "ITTFL", "SAFFL", "STRATA2")
+            ),
+            selected = NULL,
+            multiple = FALSE
+          )
+        )
+      ),
+      plot_options = list(
+        stacked = TRUE,
+        label_bars = FALSE,
+        rotate_bar_labels = TRUE,
+        rotate_x_label = TRUE,
+        rotate_y_label = TRUE,
+        flip_axis = TRUE,
+        show_n = FALSE
+      )
```

Comment:
```suggestion
      ),
      plot_height = c(600L, 200L, 2000L),
      plot_width = NULL,
      pre_output = NULL,
      post_output = NULL,
     ggplot2_args = teal.widgets::ggplot2_args()
```
LEts add all the default options.

Reply:
Sounds good. The other options have non-default values though as those are checked explicitly.